apps/heft/src/pluginFramework/HeftLifecycleSession.ts

```diff
@@ -162,12 +173,12 @@ export class HeftLifecycleSession implements IHeftLifecycleSession {
   /**
    * @internal
    */
-  public readonly metricsCollector: MetricsCollector;
+  public readonly _metricsCollector: MetricsCollector;
```

Comment:
The mixing of the meaning of the `_` prefix is really confusing, since the common case is it indicates "private property", yet sometimes it mean "public, but only intended for use by some consumers", which are very different things.

Reply:
We've historically used these interchangeably.